src/api/axiosInstance.ts

```
@@ -54,58 +56,44 @@ axiosInstance.interceptors.request.use(
-
-    // 구독자들에게 새 토큰 전달
-    notifyTokenRefreshed(newAccess ?? null);
 
+    window.dispatchEvent(
+      new CustomEvent('auth:tokenRefreshed', {
+        detail: { accessToken: newAccess, refreshToken: newRefresh },
+      })
+    );
+    notifyTokenRefreshed(newAccess ?? null);
     return newAccess ?? null;
   } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)